vs-15-2017-cxx17.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2015-2017, Ruslan Baratov
```

Comment:
Please add it to toolchain table:
* https://github.com/ruslo/polly/blob/b9819d2e7f2be813b97654f6cbba20831751fa58/bin/detail/toolchain_table.py#L180-L185